src/applications/static-pages/createAdditionalInfoWidget.js

```diff
@@ -0,0 +1,44 @@
+import { qS, qSA, removeChildNodes } from './helpers';
+
+export default function createAdditionalInfoWidget() {
+  const widgets = qSA(document, '.additional-info-container');
+
+  if (widgets.length) {
+    widgets.forEach(el => {
+      const titleNode = qS(el, '.additional-info-title');
+      const titleText =
+        (titleNode && titleNode.textContent) || 'More information';
+      const contentContainer = qS(el, '.additional-info-content').parentNode;
+      const contentContainerID = contentContainer.id;
+      const contentMarkup = qS(el, '.additional-info-content').innerHTML;
+
```

Comment:
@erikphansen recommended simplifying the markup and @DanielleSoCompany indicated that being able to optionally include a custom title would be helpful.